### PR TITLE
[ISSUE-336] Ability to deploy different drive managers

### PR DESCRIPTION
**Pull request by AnastasiaShemyakinskaya**
### Issue dell/csi-baremetal#336
_Describe your changes_
* Add handling of Drivemgr component of CSI deployment.
* Add different images usage depends on test environment for all node containers
## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
_Testing details_

### CODE REVIEW

**Review by safronovD — APPROVED** [review]

File: pkg/node.go [hunk 1]
```diff
@@ -144,32 +149,68 @@ func createNodeVolumes() []corev1.Volume {
...
+			lp = v
+			continue
+		}
+		if v.Name == driverRegistrarSidecar {
+			dr = v
+			continue
+		}
+	}
+	if lp == nil {
+		lp = constructSidecar(livenessProbeSidecar, csi.Spec.GlobalRegistry, "v2.1.0", "Always")
```
Comment (safronovD) [inline pkg/node.go:188]: Shouldn't we move sidecar version to constants too?

File: api/v1/components/drivemgr.go [hunk 2]
```diff
@@ -18,9 +18,7 @@ package components
...
 // DriveMgr represents drive manager node component
 type DriveMgr struct {
-	Image            *Image `json:"image,omitempty"`
-	Endpoint         string `json:"string"`
-	DeployConfig     bool   `json:"deployConfig"`
-	AmountOfLoopback uint64 `json:"amountOfLoopback"`
-	SizeOfLoopback   string `json:"sizeOfLoopback"`
+	Image        *Image `json:"image,omitempty"`
+	Endpoint     string `json:"string"`
+	DeployConfig bool   `json:"deployConfig"`
```
Comment (mishoyama) [inline api/v1/components/drivemgr.go:23]: I think deploy config is redundant and can be eliminated

**Review by mishoyama — CHANGES_REQUESTED** [review]

File: api/v1/components/drivemgr.go [hunk 3]
```diff
@@ -18,9 +18,6 @@ package components
 
 // DriveMgr represents drive manager node component
 type DriveMgr struct {
-	Image            *Image `json:"image,omitempty"`
-	Endpoint         string `json:"string"`
-	DeployConfig     bool   `json:"deployConfig"`
-	AmountOfLoopback uint64 `json:"amountOfLoopback"`
```
Comment (mishoyama) [inline api/v1/components/drivemgr.go:24]: Why these settings are removed?

Comment (AnastasiaShemyakinskaya) [reply api/v1/components/drivemgr.go:24]: These setting are configured throw ConfigMap, but it's created separately using `kubectl apply`. To deploy drivemgr we need to know only if we should use config or not. We don't really need these values for deployment.

Comment (mishoyama) [reply api/v1/components/drivemgr.go:24]: ConfigMap is deployed via helm and those parameters allow to override default values.

**Review by mishoyama — APPROVED** [review]

**Pull request merged**